src/Faker/Provider/ko_KR/Text.php

```diff
@@ -4,6 +4,13 @@
 
 class Text extends \Faker\Provider\Text
 {
+    public function realText($maxNbChars = 200, $indexSize = 2)
+    {
+        $realText = parent::realText($maxNbChars, $indexSize);
+
+        return iconv('UTF-8', 'UTF-8//IGNORE', $realText);
```

Comment:
This seems to be an admission of powerlessness. Instead of ignoring the bad characters, why not fix the bug that generates them?

Reply:
that's a good point.
To solve the problem, I will writing *realText* function for  Korean character and resending a PR.

thanks!